packages/k8s-cni/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor:
+# Maintainer:
```

Comment:
Could you become the maintainer of this package?

Reply:
Yes, sure!

Reply:
Please could you put your name/email at line 2?